mypyc/primitives/str_ops.py

```diff
@@ -73,6 +73,7 @@
             return_type=str_rprimitive,
             c_function_name='CPyStr_Append',
             error_kind=ERR_MAGIC,
+            var_arg_type=None,
```

Comment:
Minor nit: I think that this argument can be left out and we can rely on the default; (almost) all binary ops will likely use positional arguments.

Reply:
This is also a miss. It shouldn't be explicitly set